.github/actions/sign-attestations/src/main.ts

```diff
@@ -0,0 +1,53 @@
+import * as core from "@actions/core";
+import * as fs from "fs";
+import * as sigstore from "sigstore";
+import * as path from "path";
+
+const signOptions = {
+  oidcClientID: "sigstore",
+  oidcIssuer: "https://oauth2.sigstore.dev/auth",
+  rekorBaseURL: sigstore.sigstore.DEFAULT_REKOR_BASE_URL,
+};
+
+async function run(): Promise<void> {
+  try {
+    // Attestations
+    const attestationFolder = core.getInput("attestations");
+    const payloadType = core.getInput("payload-type");
+
+    // Output folder
+    const outputFolder = core.getInput("output-folder");
```

Comment:
output-folder is trusted. If not sure, we can addd a verification for dir traversal.

Reply:
Yes, it should be provided by the trusted workflow. Either way, I'll validate. Part of why I didn't add that in was because it required c/p the util. 

I'll file an issue about reducing code-dupe between GHA